resources/scripts/yanu-bootstrap-termux_proot.sh

```diff
@@ -11,20 +14,137 @@ proot() {
     echo -e "Running in PROOT:\n$*" # wtf is '$*'?
     proot-distro login ubuntu -- "$@"
 }
-
-# Setting up deps
-termux-setup-storage || err "Failed to get permission to Internal storage."
+# Interactive prompt function 
+interactive_prompt () {
+  # Fetch version lists
+yanu_versioning=$(git ls-remote --tags https://github.com/nozwock/yanu | cut -d "/" -f 3 | sort -r | sed '0,/$/s//& <---- [ Latest Version ]/') || err "Failed to fetch available yanu release list."
+yanu_versioning_alt=$(git ls-remote --tags https://github.com/nozwock/yanu | cut -d "/" -f 3 | sort -r) || err "Failed to fetch available yanu release list."
+read -rp "Do you want to install the latest version of yanu? [Y/N]" -t 15 PROMPT_CHOICE
+if [[ "$PROMPT_CHOICE" =~ ^(y|Y|yes|Yes|YES) ]];
+	then
+		proot rm -f /usr/bin/yanu /bin/yanu || err "Failed to clean up old yanu in proot."
+		proot eget https://github.com/nozwock/yanu/ --to="/usr/bin/" || err "Failed to fetch yanu binary in proot."
```

Comment:
Wdym by this? will `eget` download according to the arch?

Reply:
yup it'll Eget will automatically detect arch 
like if your release had "termux-proot-aarch64"
or "arm64" it'll always download the arm64 (if the host arch is same)

Reply:
just don't forget to put arch in your releases xd